examples/map_transformations/reprojection_different_observers.py

```diff
@@ -29,52 +32,55 @@
 plt.rcParams['figure.figsize'] = (16, 8)
 
 ######################################################################
-# Let’s download an EUV image from both AIA and STEREO A, when their
-# separation was around 90 degrees.
+# Let’s download an EUV image from both AIA and EUVI A, when the
+# two spacecraft were separated by approximately 120 degrees.
+
+euvi = (a.vso.Source('STEREO_A')&
+        a.Instrument('EUVI') &
+        a.Time('2011-11-01', '2011-11-01T00:10:00'))
 
-stereo = (a.vso.Source('STEREO_A') &
-          a.Instrument('EUVI') &
-          a.Time('2010-08-19', '2010-08-19T00:10:00'))
 aia = (a.Instrument('AIA') &
        a.Sample(24 * u.hour) &
-       a.Time('2010-08-19', '2010-08-19T00:10:00'))
-wave = a.Wavelength(17 * u.nm, 18 * u.nm)
+       a.Time('2011-11-01', '2011-11-02'))
+
+wave = a.Wavelength(19.5 * u.nm, 19.5 * u.nm)
 
-res = Fido.search(wave, aia | stereo)
+res = Fido.search(wave, aia | euvi)
 files = Fido.fetch(res)
 
 ######################################################################
-# Create a map for each image.
+# Create a map for each image, after making sure to sort by the
+# appropriate name attribute (i.e., "AIA" and "EUVI") so that the
+# order is reliable.
 
-map_aia, map_stereo = sunpy.map.Map(sorted(files))
+map_list = sunpy.map.Map(files)
+map_list.sort(key=lambda m: m.detector)
+map_aia, map_euvi = map_list
```

Comment:
This approach is a little fancy, of course, but it seems like the most robust approach (i.e., it doesn't depend on filenames).